XPlanning/data/mobilerobot/missions/mission0.json

```diff
@@ -24,7 +24,7 @@
 		"objective": "energyConsumption",
 		"scaling-const": 0.3,
 		"min-step-value": 0.0,
-		"max-step-value": 2525
+		"max-step-value": 4185.0
```

Comment:
This needs to change to cater for added energy along the longest path if the headlamp was turned on.